anchore_engine/services/policy_engine/engine/vulns/providers.py

```diff
@@ -266,6 +281,13 @@ def delete_image_vulnerabilities(self, image: Image, db_session):
         """
         ...
 
+    @abstractmethod
+    def get_gate_utils(self) -> GateUtilProvider:
```

Comment:
Not sure if i should change this to `get_gate_util_provider()`. There is another method in this ABC that I created previously called `get_sync_utils()`, so I was following that naming convention.

Reply:
I like the get_gate_util_provider as the name is more explicit.

Reply:
Ok. Going to rename both. Will add the rename for get_gate_utils as a fixup in the first commit. Will add a new commit for renaming get_sync_utils.

Reply:
I am also +1 for naming to be more explicit.  The plurality of the method names would be more suited to returning a collection of something.   I would also change the "get_sync_utils" method to "get_sync_util_provider".